.gitlab-ci.yml

```diff
@@ -240,7 +240,7 @@ count_amis:
 check_merge_labels:
   #Build docker images if it's needed. Check if the PR has the labels associated with the image build.
   image: registry.ddbuild.io/ci/libdatadog-build/ci_docker_base:67145216
-  tags: ["runner:docker"]
+  tags: ["arch:amd64"]
```

Comment:
I think we should test this before merge. The problem: this job is only to be executed on the main branch. I think we should change temporary this, to be able to run on this PR. But, warning! if we do that we are going to push the images, we can temporary remove the --push argument, and least check that the build is ok

Reply:
As far as I can tell the job calls a wrapper scritps around `./utils/build/build_tracer_buddies.sh` and `./utils/build/build_python_base_images.sh`. Those two scripts are covered by the `generate_system_tests_end_to_end_images` job so we should be good.
Let me know, if you still want to go with the safe approach of removing the push to test out and re-add in a follow-up PR.